src/Evolution/Systems/Cce/NewmanPenrose.hpp

```diff
@@ -69,6 +69,65 @@ struct VolumeWeyl<Tags::Psi0> {
       const Scalar<SpinWeighted<ComplexDataVector, 0>>& one_minus_y);
 };
 
+/*!
+ * \brief Compute the Weyl scalar \f$\Psi_1\f$ in the volume according to the
+ * standard set of Newman-Penrose vectors.
+ *
+ * \details Our convention is \f$\Psi_1 =
+ * l^\alpha n^\beta l^\mu m^\nu C_{\alpha
+ * \beta \mu \nu}\f$.
+ *
+ * \f{align*}{
+ * \Psi_1 =
+ * &\frac{(1-y)^2}{\sqrt{2048} \sqrt{1 + K} R^2}\Bigg\{
+ * 4 J(\bar{\eth }\beta + \tfrac{1}{2} \bar{Q})
+ * - 4(1 + K) (\eth \beta + \tfrac{1}{2} Q) \\
+ * &+(1-y)\Bigg[
+ * 4 (1 + K)\eth\partial_{y}\beta -4 J\bar{\eth }\partial_{y}\beta
+ * + 4 \left(- J \frac{\bar{\eth }' R}{R} + (1 + K) \frac{\eth ' R}{R}\right)
```

Comment:
This notation is coming from @moxcodes's Mma notebook, and it distinguishes $\eth$ in Bondi-like coordinates from $\breve\eth$ in numerical coordinates. Now that I think about it, it's possible that every $\eth$ in this expression should actually be a $\breve\eth$... Does anybody know off the top of their head? Maybe @Sizheng-Ma?

Reply:
In the code, `Eth` refers to $\eth$, instead of $\breve{\eth}$ (100% sure).

Reply:
Just to clarify, here `Eth` means tags, rather than the operation.

Reply:
Also, please avoid using primes because that refers to Cauchy coordinates --- There are four coordinate systems, so it might be good to stick to the same notation.

Reply:
Then I'm confused about why in Jordan's paper, in Eq. (46c), he has $(\breve\eth R)/R$, whereas in the code, the quantity is named `eth_r_divided_by_r`. Do we happen to have `\eth R = \breve\eth R`, since `R` does not depend on `y`?

Reply:
Yes, exactly. `R` (Bondi radius of the worldtube) is a surface variable. Eq. (46c) in https://arxiv.org/pdf/2007.01339.pdf shows that $\eth R=\breve{\eth} R$. Similarly, $\partial_u R = \partial_{\breve{u}}R$.

Reply:
🤦  doh, I should have figured that out earlier, it's been a long day... Thanks, @Sizheng-Ma. I'll update the documentation to get rid of the prime symbols.